statsmodels/base/tests/test_shrink_pickle.py

```diff
@@ -57,7 +57,7 @@ def test_remove_data_pickle(self):
 
         #check pickle unpickle works on full results
         #TODO: drop of load save is tested
-        res, l = check_pickle(results._results)
+        res, lbytes = check_pickle(results._results)
```

Comment:
I'm happier with l changes, although not sure worth it either.  Especially when there isn't a specific meaning to the variable.